services/vmalert/vmalert.go

```diff
@@ -20,56 +20,83 @@ package vmalert
 import (
 	"context"
 	"io/ioutil"
+	"net"
 	"net/http"
 	"net/url"
 	"path"
 	"time"
 
 	"github.com/pkg/errors"
+	prom "github.com/prometheus/client_golang/prometheus"
 	"github.com/sirupsen/logrus"
+
+	"github.com/percona/pmm-managed/utils/irt"
 )
 
 const (
 	updateBatchDelay           = 3 * time.Second
 	configurationUpdateTimeout = 2 * time.Second
 )
 
-// VMAlert is responsible for interactions with victoria metrics.
-type VMAlert struct {
-	baseURL             *url.URL
-	client              *http.Client
-	alertingRules       *AlertingRules
-	cachedAlertingRules string
+// Service is responsible for interactions with victoria metrics.
+type Service struct {
+	baseURL       *url.URL
+	client        *http.Client
+	externalRules *ExternalRules
+	irtm          prom.Collector
```

Comment:
not a request for change just a question, what does irtm mean?

Reply:
Instrumented http.RoundTripper with Metrics 😆 See https://github.com/percona/pmm-managed/blob/PMM-2.0/utils/irt/metrics.go